backend/cmd/start.go

```diff
@@ -74,10 +75,14 @@ const (
 	// defaultEtcdName is the default etcd member node name (single-node cluster
 	// only)
 	defaultEtcdName = "default"
-	// DefaultEtcdPeerURL is the default URL to listen for Etcd peers (single-node
+	// defaultEtcdPeerURL is the default URL to listen for Etcd peers (single-node
 	// cluster only)
 	defaultEtcdPeerURL = "http://127.0.0.1:2380"
 
+	// defaultEtcdAdvertiseClientURL is the default list of this member's client
+	// URLs to advertise to the rest of the cluster
+	defaultEtcdAdvertiseClientURL = "http://localhost:2379"
```

Comment:
This makes sense to get certs working.

We have been trying to avoid the use of localhost anywhere because it can resolve to the IPv6 loopback address on some platforms. When localhost is used by a listener, this can mean that nothing is bound to IPv4. When localhost is used by a client, the endpoint it's connecting to may not be bound to IPv6.

We'll need to make sure to document troubleshooting steps for cases where this could happen.

Reply:
I think so long as etcd is listening on `[::]` by default (which should bind to both IPv4 and IPv6 loopbacks) this shouldn't actually be an issue.

Reply:
For reference, this is etcd's default too (https://coreos.com/etcd/docs/latest/op-guide/configuration.html) but yeah, we definitely need to add a mention about that in our docs